server/src/resolvers/Mutation/pyrog.ts

```diff
@@ -1,3 +1,4 @@
+import * as bcrypt from "bcryptjs";
```

Comment:
only import the symbol you need instead of *

Reply:
Fixed in cf867df